City-of-Helsinki/helsinki-paatokset
# PP-261: Compile frontpage

Made fixes to frontpage elements, adjusted some styles and tested them all together. 
Added frontpage meetings calendar as paragraph to allow ordering frontpage elements.
Modified webpack configs to match HDBT.

To test:
- Run `drush cr; drush cim -y`
- Add a new landing page. Set it as frontpage from site settings. 
- Check the sketch: https://app.abstract.com/projects/9e324ba2-c4b2-45b5-93d9-92f00c1c688e/branches/master/commits/latest/files/7bad9819-85bc-4fe1-b2bb-e53b8b35d442/layers/7F65810E-160A-4192-AA61-E88B7E0458F4?collectionId=c1d88afb-1838-4a9f-be87-d795be73bfdf&collectionLayerId=e333e93f-0539-4036-b1fe-f38c50a1cbff. Try adding appropriate paragraphs by editing the node. 
Adding the search block: tap 'hero-block' selection. Select 'search bar' as hero style.
New paragraph style has been added for frontpage calendar. It should be found in the paragraphs dropdown.
Note that news block doesn't exist yet and cannot be added.
- `cd public/themes/custom/helfi_paatokset`. Try running `npm i`, `npm run dev` and `npm run build` commands here to verify they still work.
- Click around the site and verify that styles aren't broken.
